rs/registry/canister/src/mutations/node_management/do_add_node.rs

```diff
@@ -76,11 +76,26 @@ impl Registry {
             ));
         }
 
-        // 3. Validate keys and get the node id
+        // 3. Get valid type if type is in request
```

Comment:
Just to double check:
- the initial code will tolerate `AddNodePayload`s without the `node_type` field set
- we will later make this field mandatory, once everything else in the stack is implemented and tested
?

Reply:
Yes, that's correct.  It's not required, but if it's provided, it's validated